packages/css/src/components/buttons.css

```diff
@@ -1,138 +1,140 @@
-    --button-background-color-focus: var(--vtmn-color_brand-digital);
-    --button-color: var(--vtmn-color_white);
-    --button-border-color: transparent;
+    --btn-border-color: transparent;
+
+    @apply vtmn-bg-brand-digital
+      vtmn-text-white
+      hover:vtmn-bg-brand-digital-dark-1
+      active:vtmn-bg-brand-digital-dark-2;
   }
 
   /* Primary reversed button */
   .vtmn-btn_variant--primary-reversed {
-    --button-background-color: var(--vtmn-color_white);
-    --button-background-color-hover: rgba(255, 255, 255, 0.8);
-    --button-background-color-active: rgba(255, 255, 255, 0.6);
-    --button-background-color-focus: var(--vtmn-color_white);
-    --button-color: var(--vtmn-color_black);
-    --button-border-color: transparent;
+    --btn-border-color: transparent;
+
+    @apply vtmn-bg-white
+      vtmn-text-black;
+  }
+  .vtmn-btn_variant--primary-reversed:not([disabled]):hover {
+    background-color: rgba(255, 255, 255, 0.8);
+  }
+  .vtmn-btn_variant--primary-reversed:not([disabled]):active {
+    background-color: rgba(255, 255, 255, 0.6);
   }
 
   /* Secondary button */
   .vtmn-btn_variant--secondary {
-    --button-background-color: transparent;
-    --button-background-color-hover: rgba(115, 128, 134, 0.05);
-    --button-background-color-active: rgba(115, 128, 134, 0.15);
-    --button-background-color-focus: transparent;
-    --button-color: var(--vtmn-color_black);
-    --button-border-color: var(--vtmn-color_black);
+    --btn-border-color: var(--vtmn-color_black);
+
+    @apply vtmn-bg-transparent
+      vtmn-text-black;
+  }
+  .vtmn-btn_variant--secondary:not([disabled]):hover {
+    background-color: rgba(115, 128, 134, 0.05);
+  }
+  .vtmn-btn_variant--secondary:not([disabled]):active {
+    background-color: rgba(115, 128, 134, 0.15);
   }
 
   /* Secondary reversed button */
   .vtmn-btn_variant--secondary-reversed {
-    --button-background-color: transparent;
-    --button-background-color-hover: rgba(255, 255, 255, 0.2);
-    --button-background-color-active: rgba(255, 255, 255, 0.4);
-    --button-background-color-focus: transparent;
-    --button-color: var(--vtmn-color_white);
-    --button-border-color: var(--vtmn-color_white);
+    --btn-border-color: var(--vtmn-color_white);
+
+    @apply vtmn-bg-transparent
+      vtmn-text-white;
+  }
+  .vtmn-btn_variant--secondary-reversed:not([disabled]):hover {
+    background-color: rgba(255, 255, 255, 0.2);
+  }
+  .vtmn-btn_variant--secondary-reversed:not([disabled]):active {
+    background-color: rgba(255, 255, 255, 0.4);
   }
 
   /* Ghost button */
   .vtmn-btn_variant--ghost {
-    --button-background-color: transparent;
-    --button-background-color-hover: rgba(0, 125, 188, 0.05);
-    --button-background-color-active: rgba(0, 125, 188, 0.15);
-    --button-background-color-focus: transparent;
-    --button-color: var(--vtmn-color_brand-digital);
-    --button-border-color: transparent;
+    --btn-border-color: transparent;
```

Comment:
var(--vtmn-color_transparent);

Reply:
I've decided to delete this variable thanks to @crotru comment: https://github.com/Decathlon/vitamin-web/pull/37/files#r551223862